bin/handler-zendesk.rb

```diff
@@ -82,7 +82,7 @@ def handle
         end
       end
     rescue Timeout::Error
-      puts 'zendesk -- timed out while attempting to create a ticket for #{ticket_subject} --'
+      puts "zendesk -- timed out while attempting to create a ticket for #{ticket_subject} --"
```

Comment:
Actually this is not really a fix and are equivalent just a personal preference of mine. I will remove this if you feel we should leave it alone.
```
$ irb
irb(main):001:0> a = 'foo'
=> "foo"
irb(main):002:0> a = 'bar'
=> "bar"
irb(main):003:0> p "foo'#{a}'"
"foo'bar'"
=> "foo'bar'"
irb(main):004:0> p "foo\'#{a}\'"
"foo'bar'"
=> "foo'bar'"
```
I just prefer to be explicit as to be honest I actually assumed the other did not work but I guess once you have double quotes it will honor single quoted strings that request interpolation. As the opposite does not work I had always assumed this would be an issue but looking back it actually does make sense.